test/utility/misc.c

```diff
@@ -212,7 +212,7 @@ testutil_progress(TEST_OPTS *opts, const char *message)
     uint64_t now;
 
     if (opts->progress_fp == NULL)
-        testutil_checksys((opts->progress_fp = fopen(opts->progress_file_name, "w")) == NULL);
+        testutil_assert_errno((opts->progress_fp = fopen(opts->progress_file_name, "w")) != NULL);
```

Comment:
Sorry, I don't understand why do we need to invert all the expressions? In this particular case it was `== NULL` and now it's `!= NULL`. As far as I can see we've just renamed `testutil_checksys` to `testutil_assert_errno`, no logic has been changed in that macros.

Reply:
Nevermind, I was looking at an old version.